CONTRIBUTING.md

```diff
@@ -85,7 +95,7 @@ Our goal with the changelog is to document all changes made with each version of
 When submitting a pull request, please run the `add-change` script and commit the resulting JSON file so that your change gets added to the changelog.
 From SDK root:
 ```
-node ./scripts/changelog/add-change.js
+npm run-script add-change
```

Comment:
nitpick, `npm run add-change` should work as well (and has slightly fewer characters!)